deploy/examples/common-second-cluster.yaml

```
@@ -21,7 +21,7 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-system
-    namespace: rook-ceph
+    namespace: rook-ceph # namespace:cluster
```

I believe this is the operator namespace
```suggestion
    namespace: rook-ceph # namespace:operator
```